esios/__init__.py

```diff
@@ -1,7 +1,7 @@
 try:
     VERSION = __import__('pkg_resources') \
         .get_distribution(__name__).version
-except Exception, e:
+except Exception as e:
     VERSION = 'unknown'
 
-from service import Esios
+from .service import Esios
```

Comment:
use `from __future__ import absolute_import` in the first sentence

Reply:
Not needed to work but interesting to ensure that all imports are equal-performed. Change applied